src/pages/iou/IOUDetailsModal.js

```diff
@@ -26,8 +27,8 @@ const matchType = PropTypes.shape({
 });
 
 const defaultProps = {
-    iouReport: undefined,
     iou: {},
+    iouReport: null,
```

Comment:
Thanks, undefined does the job just fine. The default prop feels unnecessary now, but I've left it in to avoid the following lint error: 
> propType "iouReport" is not required, but has no corresponding defaultProps declaratio